apps/api/src/services/github/sync.ts

```diff
@@ -105,6 +108,8 @@ export function createSyncGithubRepositories(dependencies: Dependencies) {
       await processDeleteOldEntities(databaseTime.now, syncErrors);
     }
 
+    await processDisableOldEntities(databaseTime.now, syncErrors);
```

Comment:
Can we move this line inside the if statement on top of it (if syncErrors.length === 0) ?